docs/standalone.md

```diff
@@ -0,0 +1,106 @@
+# Standalone usage
```

Comment:
Moved from docs/maintainers, expanded with the missing steps needed to actually get play-frontend-govuk working standalone.